contracts/MarketMaker.sol

```diff
@@ -32,7 +37,7 @@ contract MarketMaker is Ownable, IERC1155TokenReceiver {
     Stages public stage;
     enum Stages {
         MarketCreated,
```

Comment:
`MarketCreated` -> `MarketRunning`? Unpausing to `MarketCreated` doesn't sound right.

Also, I think I forgot to rename this when I originally changed it over. Maybe `AMMRunning`, `AMMPaused` etc.?

Maybe even better: simply use `Running`, `Paused`, etc. Code outside this contract would have to refer to this type as `MarketMaker.Stage`, and its values as e.g. `MarketMaker.Stage.Paused` anyway, so there is no ambiguity.